### ISSUE TITLE
Error with dummy variables - bug

### ISSUE THREAD

**Issue by lilyclements** [africanmathsinitiative/R-Instat#2771]
Steven and I found an interesting issue which I have not come across before. I have a dataset I've been using with the variable name "stationStation.A"
When I put this into the receiver after using dlgDummyVariables, it removes the dot and so the variable it reads in is "stationStation A", due to this space in the name it results in an error

![image](https://cloud.githubusercontent.com/assets/21180424/23893599/ff65c98a-0896-11e7-9832-12fdc00acd3f.png)

Status Error for stationStation A <- InstatDataObject$get_columns_from_data(col_names="stationStation A", data_name="wind_df")
 : unexpected symbol

**Comment by lilyclements** [comment]
So, I've tested it out and worked out how I managed to get this particular error.

I have a dataset where the variables for a factor are "Station A" and "Station B"
I opened the dummy variables dialog and inputted station. This gave the columns stationStation.A and stationStation.B

Then I opened the dialog Random Subsets and inputted stationStation.A and then the dot was removed and replaced with a space

**Comment by lilyclements** [comment]
So, I want to clarify when this error occurs.

I have a dataset where the variables for a factor are "Station A" and "Station B"

![image](https://cloud.githubusercontent.com/assets/21180424/23936203/981ab878-0948-11e7-835a-73a2de558c1a.png)

I opened the dummy variables dialog and inputted station. This gave the columns stationStation.A and 
stationStation.B

![image](https://cloud.githubusercontent.com/assets/21180424/23936219/b58ba7fa-0948-11e7-848a-ee34cf1ab8e5.png)

![image](https://cloud.githubusercontent.com/assets/21180424/23936234/c9a961f0-0948-11e7-82b7-0513fd28163b.png)

Then I opened the dialog Random Subsets and inputted stationStation.A and then the dot was removed and replaced with a space. This dot replacing with a space happens in other dialogs too once I've used dlgDummyVariables
To clarify, if I manually change a name (let's say wind_dirs) to have a dot (wind_dirs.1) then this problem does not occur so I assume it is an issue with dlgDummyVariables

![image](https://cloud.githubusercontent.com/assets/21180424/23936289/30e37efa-0949-11e7-9776-82fb4d32f851.png)
